manifest.md

```diff
@@ -60,17 +60,26 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
 
-- **`annotations`** *string-string hashmap*
+- **`annotations`** *string-string map*
 
     This OPTIONAL property contains arbitrary metadata for the manifest list.
-    Annotations is a key-value, unordered hashmap.
-    Keys are unique, and best practice is to namespace the keys.
-    Common annotation keys include:
-    * **created** date on which the image was built (string, timestamps type)
-    * **authors** contact details of the people or organization responsible for the image (freeform string)
-    * **homepage** URL to find more information on the image (string, must be a URL with scheme HTTP or HTTPS)
-    * **documentation** URL to get documentation on the image (string, must be a URL with scheme HTTP or HTTPS)
-
+    Annotations MUST BE a key-value map where both the key and value MUST be strings.
+    While the value MUST be present, it MAY be an empty string.
+    Keys MUST be unique within this map, and best practice is to namespace the keys.
+    Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
+    Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
+    If there are no annotations then this property MAY either be absent or be an empty map.
+    Implementations that are reading/processing manifest lists MUST NOT generate an error if they encounter an unknown annotation key.
+
+	This specification defines the following keys, which MAY be used by manifest list authors:
```

Comment:
On Wed, Sep 07, 2016 at 08:36:40AM -0700, Jonathan Boulle wrote:
> could we put it in a common section and reference it from both the
> manifest list and the image manifest? e.g. "well known
> opencontainers keys" or something

I think we want to put it in the lowest type (manifest?  config?) and
have other types link there [1].

[1]: https://github.com/opencontainers/image-spec/pull/164#issuecomment-243952187